tests/panels/charts/gauge/test_gauge.py

```
@@ -0,0 +1,266 @@
+from dashboard_compiler.panels.charts.gauge.compile import (
+    compile_esql_gauge_chart,
+    compile_lens_gauge_chart,
+)
+from dashboard_compiler.panels.charts.gauge.config import ESQLGaugeChart, LensGaugeChart
+
+
+def compile_gauge_chart_snapshot(config: dict[str, Any], chart_type: str = 'lens') -> dict[str, Any]:
+    """Compile gauge chart config and return dict for snapshot testing."""
+    if chart_type == 'lens':
+        lens_chart = LensGaugeChart.model_validate(config)
+        layer_id, kbn_columns_by_id, kbn_state_visualization = compile_lens_gauge_chart(lens_gauge_chart=lens_chart)
```

Ruff (0.14.10)

Unpacked variable `kbn_columns_by_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)